### PR TITLE
SQL Model table name update

### LINKED ISSUE #137: What's new: Updated SQL Table names
Default tables have been updated to make the model easier to understand
All tables are names have the structure:

```
a) [model]
b) [model]_subtype
c) [model]_[model]
d) [model]_[model]_subtype
 ```
Where **model** is the datatype. 
Singular names are used, when a plural name shows up - it means that it holds relations to many tables.

### PR DESCRIPTION
**Pull request by Mauriceac**
- SQL Data Model documentation revised and edited.
- SQL model name update.

closes #137

### CODE REVIEW

**Comment by Mauriceac** [comment]
## OVERVIEW PDF PREVIEW
[Business Intelligence & SQL _ OVERVIEW.pdf](https://github.com/Cotalker/documentation/files/6611799/Business.Intelligence.SQL._.OVERVIEW.pdf)
THIS MODIFIES: https://doc.cotalker.com/docs/documentation/sql_bi/overview

--------

## SQL DATA MODEL
[SQL Data Model _ Partner & Technical Consultants Documentation.pdf](https://github.com/Cotalker/documentation/files/6611812/SQL.Data.Model._.Partner.Technical.Consultants.Documentation.pdf)
THIS MODIFIES: https://doc.cotalker.com/docs/documentation/sql_bi/model

**Comment by acedward** [comment]
Revisando

**Comment by Mauriceac** [comment]
## Update Blog Draft:

Screen Shot 2021-06-08 at 1 50 05 PM

**Comment by acedward** [comment]
@Mauriceac 
image

All these tables where reanmed cu_*  --> channel_user

File: docs/documentation/sql_bi/model.md [hunk 1]
```diff
@@ -1,316 +1,362 @@
...
-createdat         timestamp,
-startdate         timestamp,
-enddate           timestamp,
-modifiedat        timestamp,
-modifiedat_unixms bigint,
-__DYNAMIC__         __DYNAMIC__,
-uuid              varchar(1000) not null primary key
+```json
+cot_user:           varchar(24),
+channel:            varchar(24),
```
[the remark this answers is not in the document]
Comment (Mauriceac) [reply docs/documentation/sql_bi/model.md:105]: @acedward It seems like a good idea, but from what I see from Metabase the columns have these names i.e., "cot_user" and "channel". Wouldn't this be confusing?

[the remark this answers is not in the document]
Comment (acedward) [reply docs/documentation/sql_bi/model.md:105]: yes 100% my bad

File: docs/documentation/sql_bi/model.md [hunk 2]
```diff
@@ -1,316 +1,362 @@
...
-value      varchar(1000)
-```
 
 
-## sp_* {#sp_}
-Survey Property Relations  
-  
+------
+
+## sp_* {#sp}
```
Comment (acedward) [inline docs/documentation/sql_bi/model.md:178]: survey property model

File: docs/documentation/sql_bi/model.md [hunk 3]
```diff
@@ -1,316 +1,362 @@
...
+device:             varchar(1000),
+createdat:          timestamp,
+endedat:            timestamp,
+modifiedat_unixms:  bigint
 ```
 
 
+------
+
 ## sl {#sl}
```
Comment (acedward) [inline docs/documentation/sql_bi/model.md:162]: survey list

**Comment by acedward** [comment]
@Mauriceac sorry about the lazy review - there are no more ab_ or cc types, all tables have full-descriptive names!

**Pull request merged**